src/browser/BrowserSettingsWidget.cpp

```diff
@@ -158,6 +158,18 @@ void BrowserSettingsWidget::loadSettings()
     m_ui->browserGlobalWarningWidget->setCloseButtonVisible(false);
     m_ui->browserGlobalWarningWidget->setAutoHideTimeout(-1);
 #endif
+#ifdef KEEPASSXC_DIST_FLATPAK
+    // Guarantees proxy path works with different flatpak installations
+    m_ui->updateBinaryPath->setChecked(true);
+    m_ui->updateBinaryPath->setEnabled(false);
+    // The sandbox makes custom proxy locations very unintuitive
+    m_ui->useCustomProxy->setChecked(false);
+    m_ui->useCustomProxy->setEnabled(false);
+    m_ui->useCustomProxy->setVisible(false);
+    m_ui->customProxyLocation->setVisible(false);
+    // Won't work with xdg portals and executables that must be browser accessible
+    m_ui->customProxyLocationBrowseButton->setVisible(false);
```

Comment:
Maybe just hide the options instead of greying them out.

Reply:
@phoerious not sure I get what you're saying. Isn't `setVisible(false)` the same as `hide()`? The only option that's not hidden with this change is `updateBinaryPath`.

Reply:
This looks correct to me, no action here

Reply:
I somehow overlooked the setVisible and only saw the setEnable, which is somewhat redundant.

Reply:
@phoerious your call, I can remove the `setEnable`s if you'd like

Reply:
It's fine. Leave it.